### PR TITLE
Created base for PlayableCharacter class

### PR DESCRIPTION
**Pull request by AlejandroWoloch**

### CODE REVIEW

File: src/PlayableCharacter.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+public abstract class PlayableCharacter extends Character{
+
+	//Atributes
+	private static byte nextlevelXP = 100; 	//Used for levelingUp
+	private int experience;					//Current XP, can't exceed experience for next level.
+	private Weapon equipedWeapon;
+	private Armor equipedArmor;
+
```
Comment (rightonhana) [inline src/PlayableCharacter.java:10]: Remember to add the getters and setters, override toString and equals methods (use super method in the override)

File: src/PlayableCharacter.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+public abstract class PlayableCharacter extends Character{
+
+	//Atributes
+	private static byte nextlevelXP = 100; 	//Used for levelingUp
```
Comment (ducampcarlos) [inline src/PlayableCharacter.java]: Why is this attribute static?
I think this shouldn't be static

File: src/PlayableCharacter.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	private int experience;					//Current XP, can't exceed experience for next level.
+	private Weapon equipedWeapon;
+	private Armor equipedArmor;
```
Comment (ducampcarlos) [inline src/PlayableCharacter.java]: We're not going to use items (for now), so this 2 attributes should be commented or removed

**Comment by rightonhana** [comment]
:shipit: lgtm

**Comment by ducampcarlos** [comment]
Looks good

**Pull request merged**